ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/DepositsValidatorTest.java

```diff
@@ -59,7 +122,34 @@ public void invalidateProhibitedDepositsRoot() {
 
   @Test
   public void validateAllowedDeposits() {
-    // TODO 6110: Validate deposits in AllowedDeposits to be included in next PR
+    final BlockDataGenerator.BlockOptions blockOptions =
+        BlockDataGenerator.BlockOptions.create()
+            .setDeposits(Optional.of(List.of(DEPOSIT_1, DEPOSIT_2)));
+    final Block block = blockDataGenerator.block(blockOptions);
+
+    final TransactionReceipt receipt =
+        new TransactionReceipt(null, 0L, List.of(LOG_1, LOG_2), Optional.empty());
+
+    assertThat(new DepositsValidator.AllowedDeposits().validateDeposits(block, List.of(receipt)))
+        .isTrue();
+  }
+
+  @Test
+  public void validateAllowedDeposits2() {
```

Comment:
Please can you use a more descriptive name to distinguish what each test is doing